Documentation/.readthedocs.yaml

```diff
@@ -0,0 +1,22 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+# Build documentation with Sphinx
+sphinx:
+  configuration: Documentation/conf.py
```

Comment:
Should it be also here the full path?

Reply:
https://docs.readthedocs.io/en/stable/config-file/v2.html

> This file is named .readthedocs.yaml and should be placed in the top level of your Git repository.

Reply:
> This file is named .readthedocs.yaml and should be placed in the top level of your Git repository.

Yes!

Reply:
Apologies, I was wrong on this one. Logging to RTD interface, I see this is configurable:

![image](https://github.com/cilium/cilium/assets/17001771/3d2af176-f9a7-4e31-a7df-eb4627346c28)

and it's under `Documentation/`. @aanm did you just configure that? Do we have a preferred location for this file?

Reply:
Yes, I configured that to be the same path as this PR so we can test this changes. The previous changes failed the bailed that's why I suggested to change the `requirements:` field

Reply:
Let's keep it like this, I'll update the path on RTD. Thanks!

Reply:
Sure! It's done.